### DISCUSSION TITLE
The ord index encounters an error and restarts every time it reaches the 840000 block height。ord索引出错，每次索引到840000区块高度时就重新开始

### DISCUSSION THREAD

**Discussion by senyi123-tech** [ordinals/ord#3637] [Q&A]
I downloaded all the data for the next BTC full node, and I also downloaded the latest version of ord. However, every time I try to index with ord, it restarts the indexing process when it reaches the 840,000 block height.
我下次好了btc全节点的所有数据，并下载了ord的最新版本，进行ord索引的时候，每次到了840000区块高度就重新开始索引。

**Comment by romanlyly** [comment]
Me too, please help
